modules/compliance-tests/src-ce2/smithy4s/compliancetests/Compat.scala

```diff
@@ -16,16 +16,19 @@
 
 package smithy4s.compliancetests
 
-import cats.effect.IO
+import cats.effect._
 import cats.effect.concurrent.Deferred
-import cats.effect.ContextShift
-import cats.effect.Timer
-import cats.effect.Concurrent
+import cats.effect.implicits._
+import cats.Monad
+import scala.concurrent.duration.FiniteDuration
 
 private[compliancetests] class CompatEffect[F[_]](implicit
-    val F: Concurrent[F]
-) {
+    val _Concurrent: Concurrent[F],
+    val _Time: Timer[F],
+    val _Monad: Monad[F]
```

Comment:
Concurrent implies Monad so you can drop Monad from this

Reply:
yeah, I did not know if it was expected we rely on type class hierarchy like that or if it was better to be explicit
I suppose I already rely on the hierarchy w/o knowing it so...